python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +762,154 @@ def run(self, inputDefects, camera):
+
+
+class MergeDefectsCombinedConnections(MergeDefectsConnections,
+                                      dimensions=("instrument", "detector")):
+    inputDefects = cT.Input(
+        name="singleExpDefects",
+        doc="Measured defect lists.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector",),
+        multiple=False,
+    )
+    secondaryInputDefects = cT.Input(
+        name="AdditionalSingleExpDefects",
+        doc="Additional measured defect lists.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector",),
+        multiple=False,
+    )
+    camera = cT.PrerequisiteInput(
+        name='camera',
+        doc="Camera associated with these defects.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
+    )
+
+    mergedDefects = cT.Output(
+        name="defects",
+        doc="Final merged defects.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector"),
+        multiple=False,
+        isCalibration=True,
+    )
+
+
+class MergeDefectsCombinedTaskConfig(MergeDefectsTaskConfig,
+                                     pipelineConnections=MergeDefectsCombinedConnections):
+    """Configuration for merging defects from combined exposure.
+    """
+    def validate(self):
+        super().validate()
+        if self.combinationMode != 'OR':
+            raise ValueError("combinationMode must be 'OR'")
+
+
+class MergeDefectsCombinedTask(MergeDefectsTask):
+    """Task to measure defects in combined images."""
+
+    ConfigClass = MergeDefectsCombinedTaskConfig
+    _DefaultName = "cpDefectMergeCombined"
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        inputs = butlerQC.get(inputRefs)
+        # Turn inputDefects and secondaryInputDefects into a list
+        # which is what MergeDefectsTask expects.
+        tempList = [inputs['inputDefects'], inputs['secondaryInputDefects']]
+        # Rename inputDefects
+        inputsCombined = {'inputDefects': tempList, 'camera': inputs['camera']}
+
+        outputs = super().run(**inputsCombined)
+        butlerQC.put(outputs, outputRefs)
+
+
+class MeasureDefectsCombinedConnections(MeasureDefectsConnections,
+                                        dimensions=("instrument", "detector")):
+    inputExp = cT.Input(
+        name="defectExps",
+        doc="Input ISR-processed combined exposures to measure.",
+        storageClass="Exposure",
+        dimensions=("instrument", "detector"),
+        multiple=False,
+        isCalibration=True,
+    )
+    camera = cT.PrerequisiteInput(
+        name='camera',
+        doc="Camera associated with this exposure.",
+        storageClass="Camera",
+        dimensions=("instrument", ),
+        isCalibration=True,
+        lookupFunction=lookupStaticCalibration,
+    )
+
+    outputDefects = cT.Output(
+        name="combinedExpDefects",
```

Comment:
This name should be the default instead of in the pipeline (and should match `cpPartialDefectsFromDarkCombined` or whatever we settle on).

Reply:
This still needs to be updated.

Reply:
Yes, thanks! I'm working on it.